assisted-events-scrape/workers/cluster_events_worker.py

```diff
@@ -244,6 +251,10 @@ def _enrich_cluster(self, cluster: dict):
         for host in cluster["hosts"]:
             reshape_host(host)
 
+    def _is_blacklisted(self, cluster: dict) -> bool:
+        if "name" in cluster and cluster["name"] in self._blacklisted_names:
+            return True
```

Comment:
```suggestion
        return "name" in cluster and cluster["name"] in self._blacklisted_names:
```
otherwise it doesn't return if it's `False`

Reply:
Ooops...